### ISSUE TITLE
"data-scrapbook-create=" incorrect time

### ISSUE THREAD

**Issue by L-uk** [danny0838/webscrapbook#237]
In the "index.html" file, if opened by a text editor the date shown in the first line is off by exactly 2 hours for me. In options I have file name set to "%create-F%_%create-H%%create-M%%create-S%_%TITLE%" the file will have the correct date, but not the html file itself. This will cause confusion, how do I solve this?

**Comment by danny0838** [comment]
I don't get you. You say that "the file has correct date" but "the HTML file doesn't have correct date"? What does "the file" mean if it's not the HTML file?

**Comment by L-uk** [comment]
Sorry if I explained it bad, I attached an image this time. The "index.html file is the one extracted from the ".htz" which I opened in a text editor. The difference is in time is "\~03~" (correct) and "
\~01~" (incorrect).
![image](https://user-images.githubusercontent.com/77592542/135876162-2502428e-ce78-4b6b-abcf-38accdf8d61a.png)

**Comment by danny0838** [comment]
The `data-scrapbook-create` attribute uses UTC time, which corresponds to `%create-F-utc%%create-H-utc%%create-M-utc%%create-S-utc%%TITLE%`.

**Comment by L-uk** [comment]
> The `data-scrapbook-create` attribute uses UTC time, which corresponds to `%create-F-utc%%create-H-utc%%create-M-utc%%create-S-utc%%TITLE%`.

Can I change what time the data-scrapbook-create attribute uses? I tried changing "Filename to save:" to "%create-F-cest%..." nothing seemed to changed.

**Comment by danny0838** [comment]
No. This field is design for better interchangeability and not intended for human reading. Changing this will break compatibility.

If you have the backend server configured, the create time will be managed otherwhere and you don't need to read it directly. If you don't use backend server and want this information, you can check the "insert infobar" option and the captured page will show an infobar  with a human-readable time stamp.
